pkg/providers/snow/snow_test.go

```diff
@@ -977,18 +977,7 @@ func TestUpgradeNeededMachineConfigChanged(t *testing.T) {
 	tt.kubeconfigClient.EXPECT().
 		Get(
 			tt.ctx,
```

Comment:
I think this will normally call kubectl the 3 times

Reply:
only once. 3 times need `Times(3)`